---
bool valid
sensor_msgs/Image frame